.github/workflows/build.yml

```diff
@@ -0,0 +1,30 @@
+name: build
+on: [push, pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        rails-version: [6, 7]
+    env:
+      BUNDLE_GEMFILE: gemfiles/rails${{ matrix.rails-version }}.gemfile
+      CS_IDP_CLIENT_SECRET: ${{ secrets.CS_IDP_CLIENT_SECRET }}
+      CS_WORKSPACE: ${{ secrets.CS_WORKSPACE }}
+      CS_KMS_KEY_ARN: ${{ secrets.CS_KMS_KEY_ARN }}
+      CS_NAMING_KEY:  ${{ secrets.CS_NAMING_KEY }}
```

Comment:
These aren't secrets, and putting them in as secrets makes it *really* difficult to "reverse engineer" what the action is doing.

Reply:
I'm going to merge as-is to get this PR landed.

I'm also not sure we really want those values as plain text in the repo even if they technically aren't secrets.

Reply:
If it turns out to be a problem we can fix it then